virttest/qemu_monitor.py

```diff
@@ -22,13 +23,23 @@
     logging.warning("Could not import json module. "
                     "QMP monitor functionality disabled.")
 
-from . import passfd_setup
 from . import utils_misc
 from . import cartesian_config
 from . import data_dir
 
 from virttest.qemu_capabilities import Flags
 
+if sys.version_info[0] >= 3.0:
```

Comment:
well technically you are comparing int to float, but let's just leave it as is...